.gitignore

```diff
@@ -36,5 +36,5 @@ gradle-local.properties
 
 # Only include main and test from src dir (for now)
 src/*
-!src/main/*
-!src/test/*
\ No newline at end of file
+!src/main/
```

Comment:
Is this related to the proposed change?

Reply:
not really
This could be added to another PR. This is mainly to ensure that only the generated classes are excluded

Reply:
Reading the exclusion example from https://git-scm.com/docs/gitignore
This seems to be the correct pattern to not exclude ``src/main``

Reply:
> It is not possible to re-include a file if a parent directory of that file is excluded.

Reply:
(i.e. I think this fix might indeed successfully work around the previous problem)